catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/formulaeditor/utils/FormulaEditorWrapper.java

```diff
@@ -110,7 +110,7 @@ public FormulaEditorWrapper performEnterString(int stringResourceId) {
 
 	public FormulaEditorWrapper checkShows(String expected) {
 		onView(FORMULA_EDITOR_TEXT_FIELD_MATCHER)
-				.check(matches(withText(expected)));
```

Comment:
string.trim() would also cover this by removing trainling and leading whitespaces.
eg. .check(matches(withText(expected.trim())));